### ISSUE TITLE
Generation of the Serenity emailable reports

### ISSUE THREAD

**Issue by timothyjmtan** [serenity-bdd/serenity-core#1520]
Hi there.

I'm currently using 2.0.27.

Regarding the emailable reports:
https://serenity-bdd.github.io/theserenitybook/latest/extended-reports.html

After setting the dependency in the pom.xml as instructed in the above documentation, and triggering the aggregate lifecycle, the only .html file I see generated is the usual acceptance report (index.html).

Is the emailable report supposed to be generated alongside the acceptance report, and what should the generated filename be?

**Comment by timothyjmtan** [comment]
Got it. Mistake on my part, have fixed and its showing. Thanks!

**Comment by Upasana1997** [comment]
Hey timothyjmtan, what was the issue, becuase i am seeing the same issue and not able to figure it out.

**Comment by jmsc7git** [comment]
Hello. I too am not seeing the serenity-summary.html file. My pom file entries....

<serenity.version>1.1.36</serenity.version>
<plugin>
	<groupId>net.serenity-bdd.maven.plugins</groupId>
	<artifactId>serenity-maven-plugin</artifactId>
	<version>${serenity.maven.version}</version>
	<dependencies>
		<dependency>
			<groupId>net.serenity-bdd</groupId>
			<artifactId>serenity-core</artifactId>
			<version>${serenity.version}</version>
		</dependency>
		<dependency>
			<groupId>net.serenity-bdd</groupId>
			<artifactId>serenity-emailer</artifactId>
			<version>2.0.41</version>
		</dependency>
	</dependencies>
	<configuration>
	    <tags>${tags}</tags>
	    <reports>email</reports>
	</configuration> 
	<executions>
		<execution>
			<id>serenity-reports</id>
			<phase>post-integration-test</phase>
			<goals>
				<goal>aggregate</goal>
			</goals>
		</execution>
	</executions>
</plugin>

Excerpt of build console.....

[INFO] --- serenity-maven-plugin:1.1.36:aggregate (default-cli) @ SerenityDemo ---
[INFO] current_project.base.dir: C:\Users\automation\Desktop\Regression Workspace\SerenityDemo
Generating HTML Story Reports from C:\Users\automation\Desktop\Regression Workspace\SerenityDemo\target\site\serenity
Generating HTML Story Reports to C:\Users\automation\Desktop\Regression Workspace\SerenityDemo\target\site\serenity
Merging requirements = [Requirement{name='Health check', type='capability' parent = 'null', cardNumber='null'}, Requirement{name='Regression', type='capability' parent = 'null', cardNumber='null'}, Requirement{name='Uat', type='capability' parent = 'null', cardNumber='null'}]
Merging requirements = []
Merged requirements set = [Requirement{name='Health check', type='capability' parent = 'null', cardNumber='null'}, Requirement{name='Regression', type='capability' parent = 'null', cardNumber='null'}, Requirement{name='Uat', type='capability' parent = 'null', cardNumber='null'}]
[INFO] REPORTS GENERATED IN C:\Users\automation\Desktop\Regression Workspace\SerenityDemo\target\site\serenity
[INFO] REPORT HOME PAGE: C:\Users\automation\Desktop\Regression Workspace\SerenityDemo\target\site\serenity\index.html
[INFO] Generating release reports for: []
GENERATE CUSTOM REPORTS
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------

I am very new to Serenity BDD so be gentle..... :-)

**Comment by wakaleo** [comment]
You need all the versions of your Serenity libraries to be recent and in sync (e.g. 2.0.41)

**Comment by jmsc7git** [comment]
I plan to sign up for the BDD with Cucumber and Serenity class on serenitydojo as I am tasked with creating an automation framework for my company and I am fascinated with Serenity BDD. I have a sample Serenity project working (although very basic) based on some of the posts I found but I want an in-depth tutorial to do things the "right way". To complicate things I am somewhat new to Maven so the pom file has been a bit of a challenge. I managed to get it working by playing around with various component version numbers but have now broken it by changing the serenity version to 2.0.41 in attempts to get it to create the serenity-summary.html file. I LOVE the reports generated in Eclipse, which is one of the reasons I sold management to go with Serenity but I now realize you cannot export those cool reports so need the emailable version. I am now getting a couple pages of "Require upper bound dependencies" errors. My question.....is there a good sample pom file that has all the required dependencies/versions to use Maven to build a Serenity BDD/Cucumber project? Thank you in advance for patience with a newbie.

**Comment by jmsc7git** [comment]
OK, scratch part of that last post...I got the pom file issues worked out and now have the serenity-summary.html file! One thing that would be great on the summary report is if it broke out the individual steps so you could see which one got an error versus the overall feature.

**Comment by wakaleo** [comment]
Glad to hear you got it sorted! Do you mean having scenarios appear in the summary report as well as the features?

**Comment by jmsc7git** [comment]
So I have:
Feature: Verify the health of XYZ application
Scenario: User logs into XYZ application and sets a value on the operations page

 Given A XYZ home page
  When User logs into application
   And User navigates from Operations page to System Properties page
   And User changes a SettingKey value 
  Then User logs off application  

(Definitely a very basic first pass but baby steps...). This time when I ran it I got a server error that caused the logoff step to fail but the summary email just says the overall Feature failed without knowing which step it occurred in.  Perhaps there is more setup I can do on the report which I need to research?
